crates/solidity/inputs/language/src/dsl.rs

```diff
@@ -0,0 +1,926 @@
+use codegen_grammar::*;
+
+slang_grammar! {
+
+    config = { // Not yet used
```

Comment:
I wonder what is missing to enable reading the values from here, instead of the `grammar` crate? Is it a tiny fix? or a few days of work?
If so, I suggest removing  `config` here, until it is actually fixed/used, to prevent confusion, and have a single source of truth.

Reply:
I ran out of puff. I'll add that now.

Reply:
done